evm-contracts/src/v0.6/dev/PrepaidAggregator.sol

```diff
@@ -475,6 +477,34 @@ contract PrepaidAggregator is AggregatorInterface, Owned {
     emit OracleAdminUpdated(_oracle, _newAdmin);
   }
 
+  /**
+   * @notice allows non-oracles to request a new round
+   */
+  function startNewRound()
+    external
+    onlyAuthorizedRequesters()
+  {
+    uint32 current = reportingRoundId;
+
+    require(rounds[current].updatedAt > 0 || timedOut(current), 'Cannot start a round mid-round');
+
+    initializeNewRound(current.add(1));
+  }
+
+  /**
+   * @notice allows the owner to specify nex non-oracles to start new rounds
+   * @param _requester is the address to set permissions for
+   * @param _allowed is a boolean specifying whether they can start new rounds or not
+   */
+  function setAuthorization(address _requester, bool _allowed)
+    external
```

Comment:
Who are these requesters going to be? It's another group which will have the ability to initiate high-frequency requests, which could be expensive for whoever is paying the oracles.

Reply:
They will be rate limited by the oracle responses, but otherwise are assumed to be part of the group paying the oracles so they are only spending their own funds.